lib/utils/tests-options-validator.js

```diff
@@ -220,16 +216,23 @@ module.exports = class TestsOptionsValidator {
    * Determines if a test execution json file should be written after running a test suite
    *  @return {boolean}
    */
-  validateNoExecutionFile() {
-    const shouldExecuteOnly = this.options.noExecutionFile && this.options.loadBalance;
+  validateWriteExecutionFile() {
+    const shouldWriteExecutionFile = this.options.writeExecutionFile && this.options.loadBalance;
 
-    if (this.options.noExecutionFile && !this.options.loadBalance) {
+    if (this.options.writeExecutionFile && !this.options.loadBalance) {
       throw new SilentError(
-        'EmberExam: You must run test suite with the `load-balance` option in order to use the `no-execution-file` option.'
+        'EmberExam: You must run test suite with the `load-balance` option in order to use the `write-execution-file` option.'
       );
     }
 
-    return shouldExecuteOnly;
+    //When `--no-launch` option is passed, a value for launch in testem config is set to be string false.
+    if (this.options.launch === 'false') {
+      throw new SilentError(
+        'EmberExam: You must not use `no-launch` option in oorder to use the `write-execution`file` option.'
```

Comment:
how about: 
'EmberExam: You must not use `no-launch` with `write-execution-file` option.'